spec/version-bump-spec.js

```diff
@@ -0,0 +1,67 @@
+const { expect } = require('chai')
+const { nextVersion } = require('../script/bump-version')
+
+describe('bump-version script', () => {
+  const nightlyPattern = /[0-9.]*(-nightly.(\d{4})(\d{2})(\d{2}))$/g
+  const betaPattern = /[0-9.]*(-beta[0-9.]*)/g
+
+  it('bumps to nightly from stable', async () => {
```

Comment:
I know this might be a change in behavior, but currently, it's possible for this flow to occur when it probably shouldn't be

`stable` -> `nightly` -> `beta`

It might make sense to disallow nightly releases once stable is reached

Reply:
i agree with this as a potential option but perhaps we should shelve that and address in a later PR?